spec/lib/payment_calculator/ecf/contract/uplift_payment_calculations_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "payment_calculator/ecf/contract/uplift_payment_calculations"
+
+class DummyClass
```

Comment:
It took me a little while to work out that `DummyClass` only exists to make the `module` testable. I wonder if it's worth a short comment saying as much or whether this would be obvious to other devs more used to this way of testing ruby modules?

Reply:
I think the name perhaps could be clearer in intent. `ModuleTestHarness` or something

Reply:
It's fairly standard naming afaik. I've seen it called "DummyClass" and "DummyTestClass".

https://semaphoreci.com/community/tutorials/testing-mixins-in-isolation-with-minitest-and-rspec
https://mixandgo.com/learn/how-to-test-ruby-modules-with-rspec